css/woudc-note.css

```diff
@@ -0,0 +1,10 @@
+.woudc-note {
+  background-color: #cef !important;
+  margin-top: 4px;
+  margin-bottom: 16px;
+}
+
+.woudc-note .v-card__title {
+  padding-top: 12px;
+  padding-bottom: 0px;
+}

```

Comment:
I looked at `color="info"` and all the other built in colours you listed, and they were all really vivid and sat uncomfortably with me. Their darkness was out-of-place with how I see the page(s) being styled. The only way for a dark color like `"info"` to make sense would be white text, which I wanted to avoid as well.

Admittedly I always gravitate way too much toward very light blues and greys, which may not be what we go for with this site. And maybe when we do decide the styles it will be different than I imagine.

Reply:
As for getting rid of margins and paddings in CSS, I would be much more comfortable with `class="m*-* p*-*"` classes. Would rather do this in a follow-up like https://github.com/woudc/woudc-ui/pull/5#discussion_r437521892 as well if possible.

Reply:
Understandable for colour preference. But keep in mind:
- We can change the colour theme later for those presets and easily in one global `.sass` file, rather than jumping into each custom class and changing it.
- The default colours preset come with accessibility and proper colour contrast (more than 5:1), which is nice. Otherwise, they have to meet WCAG (AA) requirements.

Colour contrast checker tool for reference: https://webaim.org/resources/contrastchecker/

Margins and paddings: that is fine too but shouldn't be the only tool. Mainly because we have to keep in mind ofusers with larger and smaller (mobile) screen sizes, where margin and padding adjustments may not be enough to account for. I know the grid system can make our code look cumbersome with further code indents, but it will save on the trouble of adjusting later. It's also easier to adjust too for small/medium/large screen sizes (grid system divides the screen space into 12 columns for easier fractions).